packages/js/onboarding/src/components/WooPaymentsMethodsLogos/index.tsx

```diff
@@ -0,0 +1,135 @@
+/**
+ * External dependencies
+ */
+import React, { useState, useEffect } from 'react';
+import { Fragment, createElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Visa from '../../images/cards/visa';
+import MasterCard from '../../images/cards/mastercard';
+import Amex from '../../images/cards/amex';
+import Discover from '../../images/cards/discover';
+import ApplePay from '../../images/cards/applepay';
+import GooglePay from '../../images/cards/googlepay';
+import JCB from '../../images/cards/jcb';
+import WooPay from '../../images/payment-methods/woopay';
+import AfterPay from '../../images/payment-methods/afterpay';
+import Affirm from '../../images/payment-methods/affirm';
+import Klarna from '../../images/payment-methods/klarna';
+
+/**
+ * Payment methods logos
+ */
+const PaymentMethods = [
+	{
+		name: 'visa',
+		component: <Visa key="visa" />,
+	},
+	{
+		name: 'mastercard',
+		component: <MasterCard key="mastercard" />,
+	},
+	{
+		name: 'amex',
+		component: <Amex key="amex" />,
+	},
+	{
+		name: 'discover',
+		component: <Discover key="discover" />,
+	},
+	{
+		name: 'woopay',
+		component: <WooPay key="woopay" />,
+	},
+	{
+		name: 'applepay',
+		component: <ApplePay key="applepay" />,
+	},
+	{
+		name: 'googlepay',
+		component: <GooglePay key="googlepay" />,
+	},
+	{
+		name: 'afterpay',
+		component: <AfterPay key="afterpay" />,
+	},
+	{
+		name: 'affirm',
+		component: <Affirm key="affirm" />,
+	},
+	{
+		name: 'klarna',
+		component: <Klarna key="klarna" />,
+	},
+	{
+		name: 'jcb',
+		component: <JCB key="jcb" />,
+	},
+];
+
+// Maximum number of logos to be displayed on a mobile screen.
+const maxElementsMobile = 5;
+// Maximum number of logos to be displayed on a tablet screen.
+const maxElementsTablet = 7;
+// Maximum number of logos to be displayed on a desktop screen.
+const maxElementsDesktop = 10;
+// Total number of available payment methods.
+const totalPaymentMethods = 21;
```

Comment:
I think it will be useful to also mention the source of this number in the above comment. It looks like it's coming from https://woocommerce.com/document/woopayments/payment-methods? Noting that Giropay was removed from the list recently because of [this](https://support.stripe.com/questions/availability-of-giropay-june-2024-update). That leaves us with 20.

Reply:
Great catch! I have update total payment methods to 20!